authors/apps/articles/views.py

```diff
@@ -15,18 +13,21 @@
 
 from rest_framework.permissions import (IsAuthenticated,
                                         IsAuthenticatedOrReadOnly)
-from rest_framework.exceptions import NotFound, ValidationError
+from rest_framework.exceptions import NotFound
 from rest_framework import status
 from authors.apps.core.permissions import IsAuthorOrReadOnly
 from .serializers import (ArticleSerializer,
                           GetArticleSerializer, DeleteArticleSerializer,
                           RatingSerializer, CommentSerializer,
                           DeleteCommentSerializer)
 from .models import (Article, ArticleRating, Comment)
+from .pagination import CustomPagination
 
 
 class ArticleAPIView(ListCreateAPIView):
     """Creates articles and retrieves all articles"""
+
+    pagination_class = CustomPagination
```

Comment:
Articles are not ordered in any way so pagination may yield inconsistent results. How about you add an ordering for the list of articles?

Reply:
Okay, let me add that.